### PR TITLE
Param plotting fix

### PR DESCRIPTION
**Pull request by DanielJMaher**

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@Bobfrat @birdage @FBRTMaka @oceanzus Let's take a good look at this, even after the passing/failing of the Travis CI build.  A lot of funkiness happened on this.

**Comment by DanielJMaher** [comment]
@Bobfrat @birdage Passed... give it a fast review??

**Comment by birdage** [comment]
:+1:

**Comment by DanielJMaher** [comment]
@birdage @Bobfrat Can one of you merge?

**Comment by birdage** [comment]
@DanielJMaher you might wanna test it.

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -458,6 +462,77 @@ def get_uframe_stream_contents(mooring, platform, instrument, stream_type, strea
...
+        elif dpa_flag == '0' and len(parameter_ids)>0:
+            query = '?beginDT=%s&endDT=%s&parameters=%s' % (start_time, end_time,','.join(parameter_ids))
+        elif dpa_flag == '1' and len(parameter_ids)>0:
+            query = '?beginDT=%s&endDT=%s&execDPA=true&parameters=%s' % (start_time, end_time,','.join(map(str, parameter_ids)))
+
+
+        UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
+        url = "/".join([UFRAME_DATA,mooring, platform, instrument, stream_type, stream + query])
+
+        print "***:",url
```
Comment (Bobfrat) [inline ooiservices/app/uframe/controller.py]: print?

**Pull request merged**